mockserver/__init__.py

```diff
@@ -15,19 +15,20 @@ def reset(self):
         self.expectations = []
         self._call("reset")
 
-    def stub(self, request, response, timing=None):
+    def stub(self, request, response, timing=None, time_to_live=None):
         self._call("expectation", json.dumps({
             "httpRequest": request,
             "httpResponse": response,
-            "times": (timing or _Timing()).for_expectation()
+            "times": (timing or _Timing()).for_expectation(),
+            "timeToLive": (time_to_live or _Timing()).for_expectation()
```

Comment:
i don't think there should be a default, if not specified it should not be sent to the api, this will reduce the noise in the logs when debugging.

If i remember correctly, the timing has a default because MockServer's default is actually a 1 time which i think that it should be unlimited when using a `stub` which is the kind of mocking that shouldn't care whether you call it once or multiple as long as the code reacts to the content correctly.

Reply:
And instead of using the `_Timing` method, you could use a copy of the `_to_delay` function that you might call `_to_time_to_live` perhaps?